src/target/riscv/riscv.c

```diff
@@ -1481,8 +1493,7 @@ int riscv_flush_registers(struct target *target)
 	return ERROR_OK;
 }
 
-/* Convert: RISC-V hart's halt reason --> OpenOCD's generic debug reason */
```

Comment:
Can you make this comment correct instead of deleting it? Something like:
Set OpenOCD's generic debug reason from the RISC-V halt reason.

Reply:
Addressed.

Reply:
You say it's addressed but I still see the comment here. What's going on?

Reply:
If I understood correctly, the suggestion was to rephrase the comment. I did so in [defenition](https://github.com/riscv/riscv-openocd/pull/842/files#diff-b4aa16f9e42cb8f0934baa7c8e0ec9c70a369bef98b99b26ae2e896c8aa95d6aR1496-R1498) and [declaration](https://github.com/riscv/riscv-openocd/pull/842/files#diff-6b0d494208e1b1a563ca5a3ea6c4ee820028e589f3b8d69613868d9c8e169e13R398-R400). Should I remove it from the definition?

Reply:
Yes. That was sloppy reading on my part. Sorry.